xla/tools/multihost_hlo_runner/functional_hlo_runner.cc

```diff
@@ -72,6 +70,53 @@ limitations under the License.
 
 namespace xla {
 
+absl::StatusOr<std::unique_ptr<xla::PjRtClient>> GetPjRtClient(
```

Comment:
This change could be factored out?

Reply:
It's a separate commit here and it's required to write a test for the next commit.

Reply:
> It's a separate commit here and it's required to write a test for the next commit.

I thought they get squashed when merging?